client.go

```diff
@@ -708,8 +708,8 @@ func (ee *EntityErrors) Error() string {
 }
 
 // CreateScope creates a new scope
-func (c *adminClient) CreateScope(ctx context.Context, s string) error {
-	return c.connector.CreateScope(ctx, s)
+func (c *adminClient) CreateScope(ctx context.Context, s string, md *ScopeMetadata) error {
```

Comment:
In this case, why not move the scope name inside the scopeMetaData. Anyway this is a breaking change, so we can combine them.

Reply:
Good point....